src/controls/editor-mode/components/color-scheme-controls.js

```diff
@@ -153,6 +153,11 @@ const ColorSchemeControls = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [color, selectedColorIndex, update])
 
+  const getColorType = useMemo(() => {
```

Comment:
@kc-leung I don't think I understand why you needed to restrict the color type picker for Table. No matter what you choose (HEX, RGB, HSL) the base color returned is always in HEX format.